### ISSUE TITLE
ctrl+N new instance instead of replacing existing

### ISSUE THREAD

**Issue by Ingegneus** [xournalpp/xournalpp#4477]
**Is your feature request related to a problem? Please describe.**
I sometimes want to quickly have a new blank document, that I can use simulataneously with another one. currently pressing ctrl+N creates a new blank document, but it closes the one that was already open.

**Describe the solution you'd like**
open a new instance of xournalpp instead of replacing the existing one

**Comment by K4LCIFER** [comment]
Are you referring to tabs?

**Comment by Ingegneus** [comment]
not exactly. what I prefer is a new window, but I guess there could be someone that would prefer a new tab.

**Comment by hyperupcall** [comment]
Related to #3424 and #1971
